src/main/webapp/WEB-INF/classes/com/interface_adapter/signup/SignupController.java

```diff
@@ -3,19 +3,44 @@
 import com.use_case.signup.SignupInputBoundary;
 import com.use_case.signup.SignupInputData;
 
+import javax.servlet.http.HttpServletRequest;
+import java.time.LocalDateTime;
+
 public class SignupController {
     private final SignupInputBoundary userSignupUserCaseInteracter;
 
     public SignupController(SignupInputBoundary userSignupUserCaseInteracter) {
         this.userSignupUserCaseInteracter = userSignupUserCaseInteracter;
     }
 
-    public void execute(String username, String password1, String password2) {
+    public void execute(HttpServletRequest request) {
```

Comment:
Not sure if it's a good idea to pass the request through to the controller. But we can discuss later.